build/templates/_converters.py.mako

```diff
@@ -222,6 +222,36 @@ def convert_init_with_options_dictionary(values, encoding):
     return init_with_options_string
 
 
+% if config['module_name'] == 'nitclk':
+# nitclk specific converters
+def convert_to_nitclk_session_num(item):
+    '''Convert from supported objects to NI-TClk Session Num
```

Comment:
Let's remove functionality so that this converter only works for Session classes (in other words "object with a `tclk` property of type `nitclk.SessionReference`
(one way to do things)

We can add more if/when justified.

Reply:
Opened #1042 to follow up